setup.py

```diff
@@ -9,18 +9,24 @@
       license='MIT',
       packages=find_packages(),
       install_requires=[
-          'numpy',
-          'xarray>0.10.9',
+          'xarray>=0.11.0',	  
+          'pandas',	  
+          'netcdf4>=1.2.5',	  
+          'numpy>=1.11.2',
+          'dask',
+          'distributed',
+          'toolz',
+          'pytest',		
+          'fiona',
+          'cartopy',	
+          'shapely',
+          'seaborn',		  
           'matplotlib',
           'geopandas',
-          'shapely',
-          'seaborn',
-          'pandas>=0.25',
+          'pandas',
           'hs_restclient',
-          'distributed',
-          'fiona',
-          'cartopy',
-          'netcdf4'
+          'geoviews',
+          'jupyterthemes'
```

Comment:
Neither `geoviews` or `jupyterthemes` are required in pysumma - we should remove.